README.md

```diff
@@ -52,18 +52,18 @@ includes:
 parameters:
 
     paths:
-        - app
+        - app/
 
-    # The level 9 is the highest level
+    # Level 9 is the highest level
     level: 5
 
-    ignoreErrors:
-        - '#PHPDoc tag @var#'
-
-    excludePaths:
-        - ./*/*/FileToBeExcluded.php
-
-    checkMissingIterableValueType: false
+#    ignoreErrors:
+#        - '#PHPDoc tag @var#'
+#
+#    excludePaths:
+#        - ./*/*/FileToBeExcluded.php
+#
+#    checkMissingIterableValueType: false
```

Comment:
Feel that, for most Laravel applications, we should start with `checkMissingIterableValueType: false`. @canvural What's your thoughts on this one?

Reply:
> we should start with `checkMissingIterableValueType: fals`

That starts only on level 6, the above example has level 5.

Reply:
I think I agree with both of you 😄 As @szepeviktor said that check only applies for level 6 and above. So leaving it commented here would be a good idea, if anyone wants to go above level 6.